### DISCUSSION TITLE
Malloy default behavior is unexpected from a DAX user perspective

### DISCUSSION THREAD

**Discussion by djouallah** [malloydata/malloy#631] [Q&A] [answered]
using this example

```
sql: sample_source is ||
        SELECT "red" as color, 1 as Qty
        UNION ALL SELECT "black", 1
      ;;
source: sample is from_sql(sample_source)
{
  measure : Quantity is Qty.sum()


  
   Query: main_Query is {
              group_by : color
              aggregate : Quantity
              aggregate : red is Quantity { where:  color = 'red' } 
              aggregate : black is Quantity { where:  color = 'BLACK' } 
              aggregate : all is Quantity { where:  color = 'red' or  color = 'black'}                   
                             }
     
}
```
![image](https://user-images.githubusercontent.com/12554469/180347268-0a1387d6-e8de-4885-b4f1-a09338b526d6.png)

I expect the measure Black not be case sensitive
red to return 1 in both rows
all to return 2 in both rows
my expectation is the measure should ignore the group_by  by default ?

**Comment by mtoy-googly-moogly** [comment, marked as answer]
So sorry we didn't answer this earlier.

* `measure:` does not ignore the `group_by:` it "measures" the grouping.
* You can compute a measure ignoring the grouping the grouping using [Ungrouped Aggregates](https://malloydata.github.io/documentation/language/ungrouped-aggregates.html)
* String comparisons are case sensitive, we haven't addressed this yet, the workaround which I suspect you know, but I will mention here is `black is Quantity { where: UPPER(color) = 'BLACK' }`
